lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -5,6 +5,12 @@ module LinkedData
     class FindTerm
       include Qa::Authorities::LinkedData::RdfHelper
 
+      class_attribute :authority_service, :graph_service, :language_service, :language_sort_service
+      self.authority_service = Qa::LinkedData::AuthorityUrlService
+      self.graph_service = Qa::LinkedData::GraphService
+      self.language_service = Qa::LinkedData::LanguageService
+      self.language_sort_service = Qa::LinkedData::LanguageSortService
+
```

Comment:
Prefer to use class variables to identify services used by this class to allow for simpler customizations and future refactors.